File: PASCAL/Cap05/c05ex03.pas
program c05ex03;

var
  I, N, R: Integer;

begin
  I := 1;
  while not (I > 5) do
    begin
      Write('Entre um valor numerico: ');
      ReadLn(N);

      R := N * 3;

      WriteLn('Resultado = ', R);
      WriteLn;

      I := I + 1;
    end;

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: LUA/Cap05/c05ex03.lua
-- c05ex03.lua

i = 1
while not (i > 5) do
  io.write("Entre um valor numerico: ")
  n = tonumber(io.read())

  r = n * 3

  print("Resultado = " .. r)
  print()

  i = i + 1
end

print()
io.write("Tecle <Enter> para encerrar... ")
io.read()
